src/index.tsx

```diff
@@ -253,7 +253,7 @@ const main = () => {
 
     // create the graphics device
     createGraphicsDevice(canvas, {
-        deviceTypes: (url.searchParams.has('webgpu') || observer.get('enableWebGPU') ? ['webgpu'] : []).concat(['webgl2']),
+        deviceTypes: (url.searchParams.has('webgpu') || observer.get('enableWebGPU') ? ['webgpu'] : []).concat(['webgl2', 'webgl1']),
```

Comment:
webgl2 and webgl1 are automatically added as fallbacks, so all you should need here is:
```
deviceTypes: (url.searchParams.has('webgpu') || observer.get('enableWebGPU') ? ['webgpu'] : []
```

Reply:
done, thanks